plugins/env/app/controllers/environment_variable_groups_controller.rb

```diff
@@ -5,6 +5,18 @@ class EnvironmentVariableGroupsController < ApplicationController
 
   def index
     @groups = EnvironmentVariableGroup.all.includes(:environment_variables)
+
+    if project_id = params[:project_id].presence
+      @groups = @groups.joins(:project_environment_variable_groups).
+      where("project_environment_variable_groups.project_id = ?", project_id)
+    end
+
+    if deploy_group = params[:deploy_group].presence
```

Comment:
kinda misleading since it should match `all` + the "deploy groups environment" too ?